components/collector/ci/quality.sh

```diff
@@ -2,22 +2,28 @@
 
 set -e
 
-export PYTHONDEVMODE=1
-
 run () {
     header='\033[95m'
     endstyle='\033[0m'
     echo "${header}$*${endstyle}"
     eval "$*"
 }
 
-run mypy src
+spec () {
+    echo `python -c "import toml; print([spec for spec in toml.load('pyproject.toml')['project']['optional-dependencies']['tools'] if spec.split('==')[0] == \"$*\"][0])"`
+}
+
+export PYTHONDEVMODE=1
+export PIPX_HOME=.pipx
+export PIPX_BIN_DIR=.pipx/bin
+
+run pipx run `spec mypy` --python-executable=$(which python) src
```

Comment:
surprising that it is not autoresolved - is there not some kind of pipx ini/defaults file to set?

Reply:
pipx does not seem to have a config file.